src/main/java/games/strategy/triplea/delegate/Matches.java

```diff
@@ -140,6 +140,13 @@ public boolean match(final Unit unit) {
       }
     }
   };
+  public static final Match<Unit> UnitIsDefenselessTransport = new Match<Unit>() {
+    @Override
+    public boolean match(final Unit unit) {
+      final UnitAttachment ua = UnitAttachment.get(unit.getType());
+      return ua.getTransportCapacity() != -1 && ua.getIsSea() && ua.getRawDefense() == 0;
```

Comment:
I asked you to clarify a similar comment before. If there are no warships or planes, how can there be a supportAttachment?